# This file is automatically generated by Posit Publisher; do not edit.
'$schema' = 'https://cdn.posit.co/publisher/schemas/posit-publishing-record-schema-v3.json'
server_type = 'connect_cloud'
server_url = 'https://connect.posit.cloud'
client_version = '1.22.0'
created_at = ''
dismissed_at = ''
type = 'python-shiny'
configuration_name = 'csrbsr_test-K3OO'
id = '01998282-9bbb-ba17-5046-0c3d7442c2c8'
dashboard_url = 'https://connect.posit.cloud/sfanghdr/content/01998282-9bbb-ba17-5046-0c3d7442c2c8'
direct_url = 'https://01998282-9bbb-ba17-5046-0c3d7442c2c8.share.connect.posit.cloud'
logs_url = 'https://connect.posit.cloud/sfanghdr/content/01998282-9bbb-ba17-5046-0c3d7442c2c8/history'
deployed_at = '2025-09-25T16:14:02-04:00'
bundle_id = '01998282-9bbe-4645-4d03-7030e38b7da7'
files = [
  '.posit/publish/csrbsr_test-K3OO.toml',
  '.posit/publish/deployments/deployment-I6I9.toml',
  'app.py',
  'requirements.txt'
]
requirements = [
  "ipywidgets==7.8.1 # ipyleaflet's popup is broken with >8.0",
  'ipyleaflet',
  'shiny @ git+https://github.com/posit-dev/py-shiny@main',
  'shinywidgets',
  'plotly',
  'matplotlib',
  'scipy',
  'pandas',
  'faicons',
  'streamstats',
  'geopandas',
  'matplotlib',
  'pyproj',
  'python-math',
  'ezdxf'
]

[connect_cloud]
account_name = 'sfanghdr'

[deployment_error]
code = 'unknown'
message = 'Runtime error: Unable to start the app due to a runtime error. Please check the logs for more information.'
operation = 'publish/deployContent'

[configuration]
product_type = 'connect_cloud'
'$schema' = 'https://cdn.posit.co/publisher/schemas/posit-publishing-schema-v3.json'
type = 'python-shiny'
entrypoint = 'app.py'
validate = true
files = [
  '/app.py',
  '/requirements.txt',
  '/.posit/publish/csrbsr_test-K3OO.toml',
  '/.posit/publish/deployments/deployment-I6I9.toml'
]
title = 'csrbsr_test'

[configuration.python]
version = '3.9'
